warehouse/ops-tools/config-compare/pom.xml

```diff
@@ -12,6 +12,12 @@
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
         </dependency>
+        <!-- This is normally included via hadoop-common, but we have everything excluded in the warehouse parent. -->
+        <dependency>
+            <groupId>commons-collections</groupId>
+            <artifactId>commons-collections</artifactId>
+            <version>3.2.2</version>
```

Comment:
lets move this version at least into the dependency configuration at the top level

Reply:
Fixed in latest commit.